src/components/eventInformation/eventInformation.tsx

```diff
@@ -10,13 +10,15 @@ interface EventInformationProps {
   date?: string;
   time?: string;
   locations?: ILocation[];
+  address?: string | undefined;
```

Comment:
minor: `address?` sier implisitt at den er undefined. Så trenger bare skrive det som dette
```suggestion
  address?: string;
```